### DISCUSSION TITLE
Github Pages Site Stopped Working

### DISCUSSION THREAD

**Discussion by mvnd06** [community/community#22884] [Other Feature Feedback, Questions, & Ideas]
<p>Hello, I’ve had my site up for over a year now; however, the last time I went to update it, it was not there. The url is <a href="http://www.Sc0urg3r.github.io" rel="nofollow noopener">Sc0urg3r.github.io</a>, and the <a href="https://github.com/Sc0urg3r/Sc0urg3r.github.io" rel="nofollow noopener">directory</a> is still present. I also can load everything locally. Could someone please advise me on how to get my site to show again?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Hi <a class="mention" href="/sc0urg3r">@sc0urg3r</a>,</p>
<p>Thanks for being part of the GitHub Community Forum. I’ll do all I can to help.</p>
<p>After talking with another moderator, I think they found what is going on here.</p>
<p>It looks like you have some incorrect redirects happening that are resulting in a final destination of <a href="http://www.Sc0urg3r.github.io" rel="nofollow noopener">www.Sc0urg3r.github.io</a>, which doesn’t exist. Specifically, <a href="http://Sc0ourg3r.github.io" rel="nofollow noopener">Sc0ourg3r.github.io</a> is actually redirecting to <a href="http://armandraynor.com" rel="nofollow noopener">armandraynor.com</a>. You’ll need to find the IIS server for <a href="http://armandraynor.com" rel="nofollow noopener">armandraynor.com</a> and fix the redirect you have there.</p>
<p>It might be that the server is at your DNS provider and you just need to tweak a setting somewhere.</p>
<p>I hope this helps! Let us know if you have further questions.</p>
